src/aiida/orm/utils/node.py

```diff
@@ -60,8 +60,8 @@ def load_node_class(type_string):
             return Data
 
     if base_path.startswith('process'):
```

Comment:
Based on the similar code above, I think this should be:

```suggestion
    if base_path.startswith('process.'):
```